storage/s3/build.gradle

```diff
@@ -38,5 +38,8 @@ dependencies {
 
     testImplementation "org.testcontainers:junit-jupiter:$testcontainersVersion"
     testImplementation "org.testcontainers:localstack:$testcontainersVersion"
-    integrationTestImplementation "org.wiremock:wiremock:$wireMockVersion"
+
+    integrationTestImplementation("org.wiremock:wiremock:$wireMockVersion") {
+        exclude group: "org.slf4j"
```

Comment:
It screws up logging in `s3` tests otherwise. Doing in this PR because I needed it for visual validation of masking.